idseq_dag/util/validate_constants.py

```diff
@@ -1,6 +1,8 @@
 READ_LEN_CUTOFF_LOW = 50
 READ_LEN_CUTOFF_MID = 500
 READ_LEN_CUTOFF_HIGH = 10000
+# Maximum allowed line length in characters for fastq/fasta files
+MAX_LINE_LENGTH = 10000
```

Comment:
How did we get to this value?

Reply:
Yes, it is based on the read length. We need to put a reasonable limit per line to avoid corrupted files from expanding at will during a gunzip operation, and as far as I know, we currently don't support such long reads.

Reply:
I would do `MAX_LINE_LENGTH = READ_LEN_CUTOFF_HIGH` then. 
It will clear show the reason behind it...

Reply:
done